### PR TITLE
go-tools: update version to commit 89232532

### PR DESCRIPTION
**Pull request by ra1nb0w**
- use version
- add new commands

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
macOS 10.15.5 19F101
Xcode 11.5 11E608c

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [ ] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by ra1nb0w** [comment]
this port should also have gopls

File: devel/go-tools/Portfile [hunk 1]
```diff
@@ -33,25 +34,42 @@ post-extract {
...
-	
 
 destroot {
+    foreach auth $auth_cmds {
+        xinstall -W ${workpath} ${auth} ${destroot}${prefix}/bin
+    }
     foreach cmd $cmds {
-        xinstall -W ${workpath} ${cmd} ${destroot}${prefix}/bin/
+        xinstall -W ${workpath} ${cmd} ${destroot}${prefix}/bin
     }
```
Comment (ryandesign) [inline devel/go-tools/Portfile:73]: These should be done without `foreach` loops, i.e.:

```
destroot {
     xinstall -W ${workpath} {*}${auth_cmds} {*}${cmds} ${destroot}${prefix}/bin
}
```

Comment (ra1nb0w) [reply devel/go-tools/Portfile:73]: nice syntax; thank you ryan!

**Comment by ra1nb0w** [comment]
maintainer timeout. merge

**Pull request merged**